### PR TITLE
add test to github action

### PR DESCRIPTION
**Pull request by amerharb**

### CODE REVIEW

File: webapp/src/utils/adapters/TSMessageAdapter.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { MessageData } from '.';
+import { type MessageData } from '.';
```
Comment (richardolsson) [inline webapp/src/utils/adapters/TSMessageAdapter.spec.ts:1]: Why are these changes required?

File: webapp/src/utils/config.spec.ts [hunk 2]
```diff
@@ -38,10 +40,50 @@ describe('LyraConfig', () => {
...
-      expect(config.translationsPath).toEqual(
+      expect(config.projects[0].messagesPath).toEqual(
+        '/path/to/repo/subproject/locale'
+      );
+      expect(config.projects[0].translationsPath).toEqual(
         '/path/to/repo/subproject/locale'
       );
     });
+
+    it('rad more than one projects', async () => {
```
Comment (richardolsson) [inline webapp/src/utils/config.spec.ts:51]: I think this should be called "reads more than one project".

File: webapp/src/utils/config.ts [hunk 3]
```diff
@@ -29,39 +31,36 @@ const configSchema = z.object({
...
+          return new LyraProjectConfig(
+            KIND_BY_FORMAT_VALUE[project.messages.format],
+            path.join(repoPath, project.path, project.messages.path),
+            path.join(repoPath, project.path, project.translations.path)
+          );
+        })
+      );
+    } catch (e) {
+      err(`error reading ${filename} file`);
+      throw new LyraConfigReadingError(filename);
```
Comment (richardolsson) [inline webapp/src/utils/config.ts:55]: Maybe add a test for this error?

**Review by richardolsson — COMMENTED** [review]
Nice work! Some comments and questions below. 👇

File: webapp/src/utils/config.spec.ts (hunk 2 above)
Comment (amerharb) [reply webapp/src/utils/config.spec.ts]: fixed

File: webapp/src/utils/adapters/TSMessageAdapter.spec.ts (hunk 1 above)
Comment (amerharb) [reply webapp/src/utils/adapters/TSMessageAdapter.spec.ts:1]: I have a problem in the build somewhere and this fix it (can not remember now where)

File: webapp/src/utils/config.ts (hunk 3 above)
Comment (amerharb) [reply webapp/src/utils/config.ts:60]: added

**Review by richardolsson — APPROVED** [review]
Awesome! Let's merge. 💯
